corehq/motech/repeaters/static/repeaters/js/repeat_record_report_selects.js

```diff
@@ -52,7 +52,11 @@ hqDefine('repeaters/js/repeat_record_report_selects', function() {
         }
     });
 
-    $('[name="xform-ids"]').on('click', uncheckSelects());
```

Comment:
What bug is this fixing? Attaching to `DOMNodeInserted` makes me think it's related to attaching this handler to dynamically added elements, but `on` ought to do that by default.

Reply:
Looking at this again, I think this'll work if you wrap it in a document ready handler, that's where we typically attach events: 
```
$(function () {
   $('[name="xform-ids"]').on('click', uncheckSelects());
});
```

Reply:
Unfortunately this didn't work. I've also tried other similar things and the one implemented right now is the only one working.